ovon/config.py

```diff
@@ -325,7 +325,7 @@ class HabitatConfigPlugin(SearchPathPlugin):
     def manipulate_search_path(self, search_path: ConfigSearchPath) -> None:
         search_path.append(
             provider="habitat",
-            path="pkg://config/tasks/",
+            path="/nethome/asingh3064/flash/ovon/config/tasks",
```

Comment:
Why hardcoded ?

Reply:
It was not getting the path for configs present in tasks

Reply:
Outdated